src/prefect/cli/orion.py

```diff
@@ -150,6 +153,54 @@ async def start(
     app.console.print("Orion stopped!")
 
 
+@orion_app.command()
+async def status(
+    wait: bool = typer.Option(
+        False,
+        "--wait",
+        help="When the wait command is specified the health check loop runs until successful or an optional timeout value is exceeded, otherwise the check is run only once.",
```

Comment:
```suggestion
        help="When the wait command is specified, the health check loop runs until successful or an optional timeout value is exceeded. Otherwise the check is run only once.",
```